docs/Solidity.g4

```diff
@@ -306,7 +306,7 @@ assemblyBlock
   : '{' assemblyItem* '}' ;
 
 assemblyExpression
-  : assemblyCall | assemblyLiteral ;
+  : assemblyCall | assemblyLiteral | assemblyIdentifier ;
```

Comment:
Not exactly sure how these work wrt ordering, but we should maybe take care that ``x`` in ``if x`` now actually falls under ``assemblyIdentifier``, not ``assemblyCall``... maybe we should *require* the parenthesis in the ``assembly call``?
Also ``return`` in ``assemblyCall`` is not an expression... well... I guess this just needs quite some work, maybe too much to fix it in here...